ui/components/multichain/app-header/app-header.js

```diff
@@ -142,6 +150,7 @@ export const AppHeader = ({ location }) => {
                 popupStatus={popupStatus}
                 isEvmNetwork={isEvmNetwork}
                 currentNetwork={multichainNetwork}
+                networkIconSrc={networkIconSrc}
```

Comment:
If we change the `getNetworkIcon` helper with my proposal, I think we won't need that prop anymore, since the component itself will be able to use `currentNetwork` to get the corresponding icon.